cli/internal/cloudcmd/create.go

```diff
@@ -218,7 +217,6 @@ func (c *Creator) createAzure(ctx context.Context, cl terraformClient, opts Crea
 		StateDiskType:        opts.Config.Provider.Azure.StateDiskType,
 		ImageID:              opts.image,
 		SecureBoot:           *opts.Config.Provider.Azure.SecureBoot,
-		CreateMAA:            opts.Config.Provider.Azure.EnforceIDKeyDigest == idkeydigest.MAAFallback,
```

Comment:
Needs a rebase?
The option is still there on main because we don't create the MAA for TrustedLaunch. Is the removal on purpose here?